src/tour/Tour.js

```diff
@@ -0,0 +1,61 @@
+export default {
```

Comment:
how about if we move this to per-component onboarding definition files
`src/tour/namespaces.js`, `src/tour/namespace-edit.js`, ...?

this would simplify management, and how the definition files are included